### PR TITLE
Support Spark 1.3.0

### PR DESCRIPTION
**Pull request by ehiggs**
This supports Spark 1.3 and is prebuilt for Hadoop 2.3.

### CODE REVIEW

File: easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+source_urls = ['http://www.apache.org/dyn/closer.cgi/spark/%(namelower)s-%(version)s']
+
+java = 'Java'
+javaver = '1.7.0_40'
+
+dependencies = [(java, javaver)]
+
+sanity_check_paths = {
+    'files': ['bin/spark-shell'],
+    'dirs': ['bin', 'lib', 'python']
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb]: no need to check `bin` here as you already check a file inside it.

File: easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+homepage = 'http://spark.apache.org'
+description = """Spark is Hadoop MapReduce done in memory"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin-hadoop2.3.tgz']
+source_urls = ['http://www.apache.org/dyn/closer.cgi/spark/%(namelower)s-%(version)s']
+
+java = 'Java'
+javaver = '1.7.0_40'
```
Comment (boegel) [inline easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb]: no need for these, Java is only listed in one place, just collapse this:

```python
dependencies = [('Java', '1.7.0_40')]
```

File: easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+dependencies = [(java, javaver)]
+
+sanity_check_paths = {
+    'files': ['bin/spark-shell'],
+    'dirs': ['lib', 'python']
+}
+
+modextrapaths = {'PYTHONPATH': 'python'}
```
Comment (boegel) [inline easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb]: so, it depends on Python too? or are you leaving this open deliberatly?

Comment (ehiggs) [inline easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb]: Pyspark is optional but if you want to use it, PYTHONPATHH needs to have this dir in it. Hadoop is also optional but I don't bind a version.

Comment (boegel) [inline easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb]: ok

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6588a68d3be6a27f7b80 for a full test report.

File: easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+name = 'Spark'
+version = '1.3.0'
+
+homepage = 'http://spark.apache.org'
+description = """Spark is Hadoop MapReduce done in memory"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin-hadoop2.3.tgz']
+source_urls = ['http://www.apache.org/dyn/closer.cgi/spark/%(namelower)s-%(version)s']
```
Comment (boegel) [inline easybuild/easyconfigs/s/Spark/Spark-1.3.0.eb]: This results in a mirror selection page, so make it this instead:

```python
source_urls = [
    'http://apache.belnet.be/%(namelower)s/%(namelower)s-%(version)s/',
    'http://www.eu.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s/',
    'http://www.us.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s/',
]
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/45f60216818aff5b2104 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8d7f7a78bb7a2a356c93 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ehiggs!

**Pull request merged**